docs/testing.md

```diff
@@ -13,26 +13,26 @@ does not match the style requirements.
 To run ESLint by itself, you may run the lint task:
 
 ```bash
-$ yarn lint
+yarn lint
```

Comment:
Remove `$` is to simply remove some warnings for Markdown syntax. Unrelated to the main purpose of this PR.